docs/content/app-development.rst

```diff
@@ -0,0 +1,23 @@
+App Development
+===============
+
+SDK
```

Comment:
Deserves its own page. Maybe its own section

Reply:
i think its own section on this page is fine. it won't really be explaining how to build apps, but mostly link to the resources that _do_ explain it